lib/Hakyll/Web/CompressCss.hs

```diff
@@ -43,6 +45,21 @@ compressSeparators str
     stripSecond = or $ map (isOfPrefix str) $ map (\c -> c ++ " ") separators
     separators  = [" ", "{", "}", ":", ";", ",", ">", "+", "!"]
 
+-- | Compresses separators when starting inside calc().
+compressCalcSeparators :: Int -> String -> String
+compressCalcSeparators 0 str = compressSeparators str
+compressCalcSeparators depth str
+  | stripFirst = compressCalcSeparators depth (tail str)
+  | stripSecond = compressCalcSeparators depth (head str : (drop 2 str))
+  | '(' == head str = '(' : compressCalcSeparators (depth + 1) (tail str)
+  | isPrefixOf "calc( " str = compressCalcSeparators depth ("calc(" ++ (drop 6 str))
+  | isPrefixOf "calc(" str = '(' : compressCalcSeparators (depth + 1) (drop 5 str)
+  | ')' == head str = ')' : compressCalcSeparators (depth - 1) (tail str)
+  | otherwise = head str : compressCalcSeparators depth (tail str)
```

Comment:
This will lead to an exception if brackets do not match, like in `calc((2px)`. Is it a problem?